tool/download_idls.dart

```diff
@@ -0,0 +1,44 @@
+library download_idls;
+
+import 'dart:async';
+import 'dart:io';
+
+import 'src/googlesource.dart';
+import 'src/omaha.dart';
+
+main() => new IdlDownloader().downloadIdls();
+
+class IdlDownloader {
+  static const chromiumBaseUrl = 'https://chromium.googlesource.com';
+  static const chromiumVersionPrefix = '/chromium/src/+/';
+  static const idlDirs =
+      const ['chrome/common/extensions/api', 'extensions/common/api'];
+  OmahaVersionExtractor _omahaVersionExtractor;
+  GoogleSourceCrawler _googleSourceCrawler;
+  String _version;
+
+  Future downloadIdls() async {
+    _omahaVersionExtractor = new OmahaVersionExtractor();
+    _version = await _omahaVersionExtractor.stableVersion;
+    print('Downloading IDL and JSON for APIs at Chrome $_version');
```

Comment:
we should be using a logger instead of printing, I understand if that makes a better follow up cl.